node/packages/parser/src/types.ts

```diff
@@ -8,7 +8,7 @@ export interface Fee {
   value: string
 }
 
-export interface SequencedTx extends Tx {
+export interface SequencedTx extends TxBase {
```

Comment:
Shouldn't this still extend <`TxBase` | `Tx`> as the sequenced payload we are going to be ingesting client side is possibly a `UniV2Tx` or `ZrxTx` or `ThorTx` as well? Unless we can just add `TxBase` to the `Tx` union

```suggestion
export interface SequencedTx extends <TxBase | Tx> {
```

Reply:
You're right, though as we can't extend a union type using an interface, I think the best way to handle it is with a type alias, which I've committed in https://github.com/shapeshift/unchained/pull/247/commits/0b9cbee47744a18398c252dc10bcda1a33989f92:

```ts
export type SequencedTx = Tx & {
  sequence: number
  total: number
}
```

Reply:
Good point. I typically prefer interface (guess it would have been `extends TxBase, ZrxTx, ...`), but for sake of not needing to update two locations with the set of types, that works perfect. Now I think we still want to add `BaseTx` to the `Tx` union. If no contract parsers run, it would just be a `BaseTx` or standard tx returned

Reply:
Just noticed naming as well, for my ocd, can we change `TxBase` -> `BaseTx` (or what do you think of `StandardTx`?) so it matches `ZrxTx`, `ThorTx`, etc format.

Reply:
Yep, `TxBase` bugged me too, the order of specificity was inverted. `StandardTx` 👌 https://github.com/shapeshift/unchained/pull/247/commits/d61821aabe4f0fc14c11e6c0671c0918625f359b